lhapdf_pdfsetsindex.file

```diff
@@ -355,7 +355,9 @@
 81150 JR14NNLO20FF 1
 81200 JR14NLO08VF 1
 81250 JR14NLO08FF 1
-82000 LUXqed_plus_PDF4LHC15_nnlo_100 10001
+82000 LUXqed_plus_PDF4LHC15_nnlo_100 10002
+82200 LUXqed17_plus_PDF4LHC15_nnlo_100 10001
+82350 LUXqed17_plus_PDF4LHC15_nnlo_30 20001
```

Comment:
@mkirsano can you please crosscheck how many replicas are contained in these PDF sets?
10K or 20K seem to be a little bit too much... Likely is a bug

Reply:
looking at the NumMembers for each set in /cvmfs/cms.cern.ch/lhapdf/pdfsets/6.2
these lines need to be changed to
```
82000 LUXqed_plus_PDF4LHC15_nnlo_100 108
82200 LUXqed17_plus_PDF4LHC15_nnlo_100 108
82350 LUXqed17_plus_PDF4LHC15_nnlo_30 38
```